assests/javascript/index.js

```diff
@@ -111,6 +110,37 @@ async function deleteTask(id) {
   }
 }
 
+function handleStatus() {
+  const checkboxes = Array.from(document.querySelectorAll('input[type=checkbox]'));
+
+  checkboxes.forEach((checkBox) => {
+    checkBox.addEventListener("change", function (event) {
+      updateTaskStatus(event.target.parentNode.id, this.checked);
+    })
+  })
+}
+
+async function updateTaskStatus(id, isDone) {
+  try {
+    const body = JSON.stringify({ isDone });
+    const headers = { "Content-Type": "application/json" };
```

Comment:
Now it's time to put this on global scope to prevent repeating. Refactor it in other places as well.

Reply:
ohhhh yes